R/qs_ophtha.R

```diff
@@ -35,4 +35,37 @@
 #'   }
 #'
 #' @source Constructed using qs from the {pharmaversesdtm} package
+#' @details Contains a set of 29 unique Test Short Names and Test Names: \tabular{ll}{
+#'   \strong{QSTESTCD} \tab \strong{QSTEST} \cr
+#'   VFQ1  \tab Your Overall Health Is                    \cr
+#'   VFQ10 \tab Difficulty Noticing Objects to Side       \cr
+#'   VFQ11 \tab Difficulty Seeing How People React        \cr
```

Comment:
I wonder if these values are in line with CDISC controlled terminology. Maybe we should create an issue to check and update if necessary.